src/le.py

```diff
@@ -2577,6 +2644,9 @@ def process_params(self, params):
                 self.pull_server_side_config = value == "True"
             elif name == "--datahub":
                 self.set_datahub_settings(value)
+            elif name == "--multilog":
+                # multilog is only True if pathname is good
```

Comment:
The function validate_pathname is passed the pathname passed to the agent - should there be an error in the pathname, as per the rule sin the function, then an error message is displayed and the agent exits - it is only TRUE if this does not happen and the pathname is found to be good